.github/workflows/test_periodic.yml

```diff
@@ -54,33 +55,11 @@ jobs:
           docker save ovn-daemonset-f:dev > _output/image.tar
         popd
 
-    - uses: actions/upload-artifact@v3
+    - uses: actions/upload-artifact@v4
       with:
         name: test-image
         path: dist/images/_output/image.tar
 
-  k8s:
-    if: github.repository == 'ovn-org/ovn-kubernetes' || github.event_name == 'workflow_dispatch'
```

Comment:
hmm are are not running this periodic job anymore? stale?

Reply:
This job does nothing.  It sets up Go, dowloads one (outdated) KIND binary from aojea's private repo and that's all.  The job has no output, filesystem is not preserved between jobs.  And it does nothing useful.

AFAIU, ovn-kubernetes did build kubernetes from scratch in this job in the past, but it is no longer the case since https://github.com/ovn-org/ovn-kubernetes/commit/83470ca8b61dda5d49493bb8d467a2bf1ab7f1ac .